Sources/App/Services/FeedbackService.swift

```diff
@@ -0,0 +1,287 @@
+//
+// This source file is part of the ENGAGE-HF-AI-Voice open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import ModelsR4
+import Vapor
+
+
+/// Service for generating feedback based on the patient's data (vitals, symptom score, condition change)
+@MainActor
+enum FeedbackService {
```

Comment:
I would highly suggest that we explicitly pass around a `QuestionnaireResponse` type here and in the other functions, especially as we will have multiple questionnaire responses per phone number in the eventual study; I think we should ensure that we directly construct and then pass around an object between the different services.

Reply:
Thank you for the clarifications! I think it might be easier to eventually reuse elements if we rather pass around elements across boundaries rather than explicit state that is loaded from disk. I think we don't have to do this right here but it would be great to do that in a subsequent step.